proxy/executor/strategy_base.py

```diff
@@ -113,7 +113,16 @@ async def cancel(self) -> ExecTxRespCode | None:
 
     @property
     def _cu_price(self) -> int:
-        return self._ctx.cfg.simple_cu_price
+        # Legacy transaction case (both stuck and not stuck), return value from the config.
+        if self._ctx.tx_type == 0:
+            return self._ctx.cfg.simple_cu_price
+
+        # For Dynamic Gas transaction:
+        base_fee_per_gas = self._ctx.max_fee_per_gas - self._ctx.max_priority_fee_per_gas
+        assert base_fee_per_gas > 0
+        max_priority_fee_per_gas = self._ctx.max_priority_fee_per_gas
+        # TODO EIP1559: conversion to micro lamports???
+        return max(1, int(max_priority_fee_per_gas * 5000.0 / (base_fee_per_gas * self._cu_limit)))
```

Comment:
Yep, here should be the multiplication on 1'000'000.

In the bottom part of the formula, you have cu_limit which is equal to 1'400'000, and after direct division, you will get the number less than 1.

Reply:
thanks, done